CMake/SofaPython3Tools.cmake

```diff
@@ -109,10 +109,6 @@ function(SP3_add_python_module)
         PUBLIC $<INSTALL_INTERFACE:include>
     )
 
-    if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
```

Comment:
Why did you removed it? I think it is still needed, and put to public for the tests or external libraries that include/link SP3 bindings

Reply:
Yes, it is still needed: I removed it because until I saw that Bindings.Simulation doesn't link against SofaPython3::Plugin I thought all bindings would inherit the flag from it.
I correct the PR